lib/text_diff.ml

```diff
@@ -1,21 +1,19 @@
 open Types
 
-type change = { orig : string list; new_ : string list }
-type conflict2 = Atomic of change | Record of string * change list
-type t = conflict2 list String_map.t
-type printer = { same : string Fmt.t; diff : conflict2 Fmt.t }
+type conflict2 = { orig : string list; new_ : string list }
+type common = string list
+type context = Change of conflict2 | Same of common
```

Comment:
`context` is a bit confusing here as it can both be the context or the actual change. The terminology in the original `diffutils` for this was `hunk`, we could eventually re-use this name.